chore: Adding info about monitor migration (#91)
OpenZeppelin/docs: content/defender/migration.mdx

### Timeline Considerations

- Plan for adequate testing time before fully switching over
- Consider running both systems in parallel during migration
- Schedule migration during low-traffic periods if possible

---

## Monitor Migration

### From Defender Monitor to OpenZeppelin Monitor

OpenZeppelin Monitor offers similar functionality to Defender Monitor:

- **Event and function monitoring**: Monitor smart contract events and function calls
- **Custom filtering**: Write custom JavaScript, Python or Bash filters to match specific conditions
- **Multiple notification channels**: Integrate with Slack, Telegram, Discord, webhooks and emails

### Getting Started

To begin your Monitor migration:

1. **Review your existing monitors**: Export your Defender Monitor configurations using the "Download OpenZeppelin Monitor Configurations" button
2. **Import your configurations**: Use the exported configurations to recreate your monitors in OpenZeppelin Monitor
3. **Set up OpenZeppelin Monitor**: Follow the [installation guide](/monitor) to set up the Monitor.

![Defender Monitor Migration Button](/defender/monitor-migration-information.png)

> The "OpenZeppelin Monitor Configurations" button will download a .zip file containing configuration files for every monitor in your Defender account (if you don't have to download some monitor, first delete it). These configurations are ready to copy and paste directly into OpenZeppelin Monitor. Before running the monitors, make sure to review and update any placeholder values in the configuration files, such as:

Reviewer: ```suggestion
The "OpenZeppelin Monitor Configurations" button will download a .zip file containing configuration files for every monitor in your Defender account (If for any reason you don't want a specific monitor to be migrated, we recommend you delete the monitor before clicking on the OpenZeppelin Monitor Configurations button ). These configurations are ready to copy and paste directly into OpenZeppelin Monitor. Before running the monitors, make sure to review and update any placeholder values in the configuration files, such as:
```

- **SDK integration**: Easily interact with the relayer through a companion JavaScript/TypeScript SDK
- **Configurable policies**: Define and enforce network-specific policies for transaction processing

### Getting Started

To begin your Relayer migration:

1. **Review your existing relayers**: Export your Defender Relayer configurations using the "OpenZeppelin Relayer Configurations" button

![Defender Relayer Migration Button](/defender/relayer-migration-button.png)

The "Download OpenZeppelin Relayer Configurations" button will download a .zip file containing configuration files for every relayer in your Defender account (if you don't want to download some relayer, first delete it). These configurations are ready to copy and paste directly into OpenZeppelin Relayer. Before running the relayers, make sure to review and update any placeholder values in the configuration files, such as:

2. **Set up OpenZeppelin Relayer**: Follow the [installation guide](/relayer) to deploy the Relayer on your infrastructure
3. **Import your configurations**: Use the exported configurations to recreate your relayers in OpenZeppelin Relayer

> - RPC URLs and endpoint addresses

Reviewer 2: list should be at line 99, not after step 3

2. **Set up OpenZeppelin Relayer**: Follow the [installation guide](/relayer) to deploy the Relayer on your infrastructure
3. **Import your configurations**: Use the exported configurations to recreate your relayers in OpenZeppelin Relayer

4. **Test your relayers**: Verify that transactions are processed correctly before decommissioning Defender relayers
5. **Transfer funds**: Once your OpenZeppelin Relayers are fully configured and running, transfer the funds from your Defender relayers to your new OpenZeppelin Relayer addresses using the Withdraw functionality
6. **Gradually move volume**: Shift traffic from Defender relayer to OpenZeppelin Relayer incrementally

Here's a video with the process step by step: https://www.loom.com/share/cb5e0f5d8c064a71abc8c18fac273cf0

### Understanding Signer Migration

<Callout type='warn'>
**Relayer recreation is required due to AWS KMS security constraints**

> Defender Relayers use AWS Key Management Service (AWS KMS) to secure private keys. Due to AWS KMS's security model, **private keys cannot be exported** from the KMS system. This means:

Reviewer: ```suggestion
Defender Relayers use AWS Key Management Service (AWS KMS) to secure private keys. Due to AWS KMS's security model, **private keys cannot be exported** outside AWS. This means:
```
